jaeger-core/src/main/java/io/jaegertracing/propagation/Extractor.java

```diff
@@ -14,11 +14,11 @@
 
 package io.jaegertracing.propagation;
 
-import io.jaegertracing.SpanContext;
+import io.opentracing.SpanContext;
```

Comment:
semantic change. Codecs are an internal API, Tracer takes care of converting the context to Jaeger context.